esp/public/media/scripts/ajaxschedulingmodule/ESP/Cell.js

```diff
@@ -266,6 +266,27 @@ function Cell(el, section, room_id, timeslot_id, matrix) {
                     }
                 if(n_teachers == 0) return grey;
                 return this.cellColors.HSLToRGB(0,100,100 - 10 * n_teachers); // Color red based on number of teachers
+            case "num_teachers":
+                // Color cell based on the number of teachers for the section
+                var n_teachers = Math.min(section.teachers.length, 5)
+                return this.cellColors.HSLToRGB(120,100,100 - 10 * n_teachers);
+            case "num_moderators":
+                // Color cell based on the number of moderators for the section
+                var n_moderators = Math.min(section.moderators.length, 5)
+                return this.cellColors.HSLToRGB(120,100,100 - 10 * n_moderators);
+            case "num_both":
+                // Color cell based on the number of teachers and/or moderators for the section
+                var n_moderators = Math.min(section.moderators.length + section.teachers.length, 5)
+                return this.cellColors.HSLToRGB(120,100,100 - 10 * n_moderators);
+            case "mod_cats":
+                // Color cell based on the number of moderators for the section that have not specified this section's category in the moderator form
+                var n_moderators = 0;
+                for(var moderator of section.moderator_data){
+                    if(!(section.category_id in moderator.categories)){
+                        n_moderators += 1;
+                    }
+                }
+                return this.cellColors.HSLToRGB(0,100,100 - 10 * n_moderators);
```

Comment:
Something fishy is going on. It's mostly working, but I have this one class that is highlighted red and im not sure why:
![image](https://user-images.githubusercontent.com/25392075/114795714-403a1600-9d5d-11eb-8ccc-2de477ce763a.png)
![image](https://user-images.githubusercontent.com/25392075/114795892-a9ba2480-9d5d-11eb-9196-10df9293cca1.png)
(This is for Splash Fall 13 on dev3)